assimilation_code/modules/assimilation/adaptive_inflate_mod.f90

```diff
@@ -484,7 +493,8 @@ subroutine inflate_ens(inflate_handle, ens, mean, inflate, var_in, fsprd, asprd)
 
 type(adaptive_inflate_type), intent(inout) :: inflate_handle
 real(r8),                    intent(inout) :: ens(:)
-real(r8),                    intent(in)    :: mean, inflate
+real(r8),                    intent(in)    :: mean 
+real(r8),                    intent(inout) :: inflate
```

Comment:
I think this is wrong ... inflate is not modified, should stay 'intent(in)'

Reply:
Tim, you're WRONG!